third_party/2and3/markupsafe/__init__.pyi

```diff
@@ -42,7 +42,7 @@ class Markup(text_type):
     def strip(self, chars: Optional[text_type] = None) -> Markup: ...
     def center(self, width: int, fillchar: text_type = ...) -> Markup: ...
     def zfill(self, width: int) -> Markup: ...
-    def translate(self, table: Union[Dict[int, Any], text_type]) -> Markup: ...
+    def translate(self, table: Union[Mapping[int, Union[int, text_type, None]], Sequence[Union[int, text_type, None]]]) -> str: ...
```

Comment:
The return value should still be Markup. I believe mypy will accept that, since overridden methods can give a narrower return type than their base class.

Reply:
It's been `Markup` before too, so it should work. Just a copy-paste error on my part, but nice catch.